frontend/src/concepts/pipelines/kfTypes.ts

```diff
@@ -60,7 +60,7 @@ export enum StorageStateKF {
 
 export type ParameterKF = {
   name: string;
-  value: string;
+  value: string | undefined;
```

Comment:
Can we ensure this matches the kubeflow API. Otherwise assume it was correct and fix up how we deal with the value possibly being empty.